migrations/066-all-delegates-sorted.sql

```diff
@@ -0,0 +1,88 @@
+create type order_direction_type as enum ('ASC', 'DESC');
+create type delegate_order_by_type as enum ('DATE', 'MKR', 'DELEGATORS');
+create type delegate_entry as (
+  delegate character varying(66),
+  vote_delegate character varying(66),
+  creation_date timestamp with time zone,
+  expiration_date timestamp with time zone,
+  expired boolean,
+  last_voted timestamp with time zone,
+  delegator_count int,
+  total_mkr numeric(78,18)
+);
+
+-- Small function used to return the max page size for paginated endpoints
+create or replace function api.max_page_size()
+returns int as $$
+select 30
+$$ language sql stable strict;
+
+create or replace function api.delegates(_first int, order_by delegate_order_by_type default 'DATE', order_direction order_direction_type default 'DESC', include_expired boolean default false)
+returns setof delegate_entry as $$
+declare
+  max_page_size_value int := (select api.max_page_size());
+begin
+  if _first > max_page_size_value then
+    raise exception 'Parameter FIRST cannot be greater than %.', max_page_size_value
+    return;
+  else
+    return query
+      -- Merge poll votes from Mainnet and Arbitrum and attach the timestamp to them
+      with merged_vote_events as (
+        select voter, vote_timestamp
+        from (
+          select voter, timestamp as vote_timestamp
+          from polling.voted_event A
+          left join vulcan2x.block B
+          on A.block_id = B.id
+        ) AB
+        union all
+        select voter, vote_timestamp
+        from (
+          select voter, timestamp as vote_timestamp
+          from polling.voted_event_arbitrum C
+          left join vulcan2xarbitrum.block D
+          on C.block_id = D.id
+        ) CD
+      ),
```

Comment:
I noticed in migration 062 that we define a function `voted_events_merged`. Are these doing the same thing? I'm wondering if we can reuse the same query to avoid quirks in logic and redefining queries in different ways. Perhaps there's a need to do them differently though, I don't know all the context.